src/loaders/xmf_load.c

```diff
@@ -242,6 +246,11 @@ static void xmf_translate_effect(struct xmp_event *event, uint8 effect, uint8 pa
 		param |= (param << 4);
 		xmf_insert_effect(event, FX_SETPAN, param, chn);
 		break;
+
+	case 0x11:			/* Ultra Tracker retrigger */
+		/* TODO: should support the full param range, needs testing. */
```

Comment:
Probably not necessary since the UltraTracker retrigger command is `E9` with a single nibble (like in MOD but supoosedly with some differences in behavior)